tools/build.py

```diff
@@ -59,6 +60,7 @@ def add_build_args(parser):
     parser.add_argument('--jerry-cmdline', choices=['on', 'off'], default='on', help='Build jerry command line tool (default: %(default)s)')
     parser.add_argument('--static-link', choices=['on', 'off'], default='on', help='Enable static linking of jerry command line tool (default: %(default)s)')
     parser.add_argument('--link-map', choices=['on', 'off'], default='off', help='Enable the generation of a link map file for jerry command line tool (default: %(default)s)')
+    parser.add_argument('--parallel-build', action='store',type=int, default=multiprocessing.cpu_count()+1, help='Allowed jobs at once (default: %(default)s)')
```

Comment:
Also missing spaces after `,` and around `+`. Also will the help really work? it seems to me that it should fail as we have an int default value and try to place it into the help as a string.